js/adapt-assessmentArticleExtension.js

```diff
@@ -1,46 +1,41 @@
-define([
-  'core/js/views/articleView',
-  'core/js/models/articleModel',
-  './adapt-assessmentArticleView',
-  './adapt-assessmentArticleModel',
-  './assessment'
-], function(ArticleView, ArticleModel, AdaptAssessmentArticleView, AdaptAssessmentArticleModel) {
+import ArticleView from 'core/js/views/articleView';
+import ArticleModel from 'core/js/models/articleModel';
+import AdaptAssessmentArticleView from './adapt-assessmentArticleView';
+import AdaptAssessmentArticleModel from './adapt-assessmentArticleModel';
+
```

Comment:
Assessment code needs to be included.

```suggestion
import './assessment';

```

Reply:
sorry, this was my fault